src/containers/ConversionForm/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import { Upload, Select, Divider, Button } from "antd";
+import classNames from "classnames";
+import { ActionCreator } from "redux";
+import { connect } from "react-redux";
+
+import theme from "../../components/theme/light-theme.css";
+import { State } from "../../state";
+import trajectoryStateBranch from "../../state/trajectory";
+import viewerStateBranch from "../../state/viewer";
+import {
+    ReceiveFileToConvertAction,
+    SetConversionEngineAction,
+} from "../../state/trajectory/types";
+import { AvailableEngines, Template, TemplateMap } from "../../state/trajectory/conversion-data-types";
+
+import styles from "./style.css";
+import customRequest from "./custom-request";
+import { SetErrorAction } from "../../state/viewer/types";
+import { UploadFile } from "antd/lib/upload";
+
+interface ConversionProps {
+    setConversionEngine: ActionCreator<SetConversionEngineAction>;
+    conversionProcessingData: {
+        template: Template;
+        templateMap: TemplateMap;
+        preConvertedFile: string;
```

Comment:
consider renaming to `fileToConvert` to match the `receiveFiletoConvert` below.

Reply:
point taken its a bit vague, this object is both file and template though, can try to think of a more semantic name

Reply:
I only meant `preConvertedFile` --> `fileContentsToConvert` or `fileToConvert` or something like that.

Reply:
oops got it, thanks for clarifying, will change to fileToConvert